auto_round/utils.py

```diff
@@ -3042,3 +3044,27 @@ def is_diffusion_model(model_or_path: Union[str, object]) -> bool:
         return isinstance(model_or_path, pipeline_utils.DiffusionPipeline)
     else:
         return False
+
+
+def is_separate_lm_head(model: torch.nn.Module) -> bool:
+    dir_path = model.name_or_path
+    if not os.path.isdir(dir_path):
+        dir_path = download_hf_model(dir_path)
```

Comment:
Better not download the model again. Since returning a wrong answer here doesn’t cause major issues, we can simply return a placeholder answer instead.